### ISSUE TITLE
Apps crash with no error messages using pandas 1.2.3 and interpolate(method='quadratic') / Windows 10

### ISSUE THREAD

**Issue by Shadoward** [pyinstaller/pyinstaller#5656]
Hi all,
The exe created with pyinstaller 4.2 is crashing during the phase of df.interpolate(method='quadratic'). There is no error message. The python version is working fine.

Here the version install to create the exe. 
```
Windows 10
gooey                     1.0.8            py38h885f38d_1    conda-forge
numpy                     1.20.1           py38h0cc643e_0    conda-forge
pandas                    1.2.3            py38h4c96930_0    conda-forge
pyinstaller               4.2              py38hd0d6af5_1    conda-forge
pyinstaller-hooks-contrib 2020.11            pyhd8ed1ab_0    conda-forge
python                    3.8.8           h7840368_0_cpython    conda-forge
scipy                     1.6.1            py38h5f893b4_0    conda-forge
wxpython                  4.1.1            py38heb73c8a_0    conda-forge
```

Here a more detail  code:
```
    df.set_index('DateTime', inplace=True)
    df['FirstRolling'] = df['A'].rolling(window=100, center=True).mean() # Applying a moving average (1nd smoothing)
    df['FirstRolling'].iloc[0] = df['A'].iloc[0]
    df['FirstRolling'].iloc[-1] = df['A'].iloc[-1]
    df['FirstInterpolate'] = df['FirstRolling'].interpolate(method='quadratic') # Interpolate despike with quadratic function (1st interpolation)
    df['SecondRolling'] = df['FirstInterpolate'].rolling(window=int(100), center=True).mean()  # Applying a moving average (2nd smoothing)
    df['SecondRolling'].iloc[0] = df['FirstInterpolate'].iloc[0]
    df['SecondRolling'].iloc[-1] = df['FirstInterpolate'].iloc[-1]
    df['Final'] = df['SecondRolling'].interpolate(method='quadratic') # Interpolate despike with quadratic function (2st interpolation)
    df.reset_index(inplace=True)
```

**Comment by bwoodsend** [comment]
This was fixed in #5168 so just install the development version:

```
conda remove pyinstaller
pip install https://github.com/pyinstaller/pyinstaller/archive/develop.zip
```

Then rebuild but add the `--clean` option.

**Comment by Shadoward** [comment]
Perfect, thanks!
Closing the issue
